### PR TITLE
Fix alignment issues of Join Us page

### LINKED ISSUE #1298: Hero section of Join Us page is not well aligned
**Describe the bug**
Hero section of Join Us page is not well aligned both mobile and large screens.

**To Reproduce**
1. Go to https://sefglobal.org/join-us.html
2. Click on inspect elements
3. See error

**Expected behavior**
Components in the hero section should be well aligned and responsive

**Screenshots**

*Desktop view*

3dkgd1Efz1

*mobile view*

chrome_nVld9WaUMl

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser google chrome
 - Version 106

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by sathmikaDev**
## Purpose
The purpose of this PR is to fix #1298 

## Goals
fix alignment issues and make responsive, hero section of join us page

## Approach
create a new section with two DIV to put text elements and image side by side. aligned items, center on mobile devices.

### Screenshots

chrome_l91HPvjrC0

chrome_uYPaLIYBwl

  
### Preview Link
https://pr-1300-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/content/static/footer.html [hunk 1]
```diff
@@ -8,42 +8,58 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
                 </div>
                 <div class="col-lg-6 text-lg-center btn-wrapper">
                     <a target="_blank" href="https://twitter.com/goasksef"
+
```
Comment (Piumal1999) [inline assets/content/static/footer.html]: Can you remove the changes on this file because those are not related to the issue?

Comment (sathmikaDev) [reply assets/content/static/footer.html]: > Can you remove the changes on this file because those are not related to the issue?

okay sure

**Comment by Piumal1999** [comment]
Few tips to avoid conflicts and unwanted commits:
- Always keep the local `master` branch up to date with the upstream `master` branch (`sefglobal/sef-site`)
- Always use a new branch when working on issues. Use the updated master branch for creating the new branch

You can try this procedure when working on a new issue:
1. Checkout to the master branch (`git checkout master`)
2. Optional step: Reset the local master branch
3. Get a pull from upstream to make it up to date (`git pull upstream master`)
4. Create a new branch and checkout (`git checkout -b my-new-branch`)
5. Do the changes and commit
6. Push (`git push origin my-new-branch`)
Then we can repeat from step 1 when working on another issue.

**Comment by sathmikaDev** [comment]
> 

Thank you, got it🙌

File: join-us.html [hunk 2]
```diff
@@ -40,57 +40,32 @@
...
-                    <ul class="mt-2">
-                        <li>Collaboration</li>
-                        <li>Leadership</li>
-                        <li>Digital Marketing</li>
-                        <li>Business Communication</li>
-                        <li>Programming</li>
-                    </ul>
+                <!-- image div -->
+                <div class="col-md-6 mt-5 mt-md-0">
+                    <img src="assets/img/join_us/heroNew.svg" class="img-fluid w-150">
```
Comment (Piumal1999) [inline join-us.html]:
```suggestion
                    <img src="assets/img/join_us/hero.svg" class="img-fluid w-150">
```

The new image file is missing. Replace the old image with the new image. Commit that as well.

File: assets/content/static/footer.html [hunk 3]
```diff
@@ -8,7 +8,7 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
                 </div>
                 <div class="col-lg-6 text-lg-center btn-wrapper">
                     <a target="_blank" href="https://twitter.com/goasksef"
-                        class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip"
+                        class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip"
```
Comment (Piumal1999) [inline assets/content/static/footer.html]:
```suggestion
                        class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip"
```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

**Review by Piumal1999 — APPROVED** [review]
Well done @Sathmikasenadheera01 
Thank you

**Comment by sathmikaDev** [comment]
> Well done @Sathmikasenadheera01 Thank you

thank you and you're welcome!
